compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/infer-effect-dependencies/bailout-retry/error.todo-syntax.expect.md

```diff
@@ -0,0 +1,51 @@
+
+## Input
+
+```javascript
+// @inferEffectDependencies @panicThreshold(none)
+import {useSpecialEffect} from 'shared-runtime';
+
+/**
+ * Note that a react compiler-based transform still has limitations on JS syntax.
+ * We should surface these as actionable lint / build errors to devs.
```

Comment:
The fix here is to specify the dep array (for special effect, that's a second dep array), is that right?

Reply:
Yep exactly -- I'll edit the error messages to be more actionable